recipe/meta.yaml

```
@@ -30,21 +32,37 @@ outputs:
         - python
         - pip
         - setuptools
+        - wheel
       run:
         - python
+
+    {% set ignore_tests_files = "" %}
+    # ERROR tests/prof/perf - ModuleNotFoundError: No module named 'benchmarks'
+    {% set ignore_tests_files = ignore_tests_files + " --ignore=tests/prof" %}
+    {% set ignore_tests_files = ignore_tests_files + " --ignore=tests/typing/test_pyright.py" %}
+
+    # Skiping tests [not (win and py==310) and not (win and py==311)]
+    # Windows fatal exception: stack overflow. To many tests to skip one by one
+
     test:
       imports:
         - msgspec
       requires:
         - pip
         - pytest
+        - mypy
+        - attrs
+        - pyyaml
+        - tomli    # [py<311]
+        - tomli-w
+        - msgpack-python
       source_files:
         - tests
```

```suggestion
        - tests 
        # benchmarks are here for running the tests suite.
        - benchmarks
```
It will solve `ModuleNotFoundError: No module named 'benchmarks'`

---

We don't have this pkg in the channel 

`conda search benchmarks
2 channel Terms of Service accepted
Loading channels: done
No match found for: benchmarks. Search: *benchmarks*

PackagesNotFoundError: The following packages are not available from current channels:

  - benchmarks

Current channels:

  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.`

---

It's a folder of the package itself https://github.com/jcrist/msgspec/tree/0.20.0/benchmarks